www/shards/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        crystal-lang shards 0.8.1-devel v
+
+categories          www
+platforms           darwin
+supported_archs     noarch
```

Comment:
Is it truly the case that this port installs no architecture specific files at all?